src/components/MovieDetails/MovieDetails.js

```diff
@@ -92,29 +92,36 @@ class MovieDetails extends Component {
 
   render() {
     const { id, title, backdrop_path, poster_path, release_date, overview, average_rating } = this.props;
-    return (
-      <article className="movie-details-card" >
+    if (title) {
+      return (
+        <article className="movie-details-card">
         <section className="movie-images">
-          <img src={poster_path} alt={"poster for " + title}/>
-          <img src={backdrop_path} alt={"background for " + title}/>
+        <img src={poster_path} alt={"poster for " + title}/>
+        <img src={backdrop_path} alt={"background for " + title}/>
         </section>
         <section className="movie-details">
-          <section>
-            <h1>{title}</h1>
-            <h3>Average rating: {average_rating && average_rating.toFixed(1)}/10</h3>
-            <h3>{this.displayUserRatingConditional()}</h3>
-          </section>
-          <section>
-            <h3>Release date: {release_date}</h3>
-          </section>
-          <section>
-            {overview}
-          </section>
-          {this.checkIfLoggedIn()}
+        <section>
+        <h1>{title}</h1>
+        <h3>Average rating: {average_rating && average_rating.toFixed(1)}/10</h3>
+        <h3>{this.displayUserRatingConditional()}</h3>
```

Comment:
Did we remove the toFixed in the last PR? I think it sneaked back into the code.